spec/models/throttle_spec.rb

```diff
@@ -135,10 +166,42 @@
     end
   end
 
+  describe '#expires_at' do
+    before { Timecop.freeze }
+    after { Timecop.return }
```

Comment:
I like the around filter version of timecop so we don't have to worry about matching a before with an after

```suggestion
    around do |ex|
      Timecop.freeze { ex.run }
    end
```

Reply:
>I like the around filter version of timecop 

Ooh, I like it!